contractsV5/ERC20Interface.sol

```diff
@@ -0,0 +1,76 @@
+pragma solidity 0.5.9;
+

```

Comment:
if this is compiled together with 0.4.18
it will over run artifact?
right?

Reply:
Depends on the order of compilation. Currently, the v4 contracts are compiled first, then v5, so yes, this will overwrite the 0.4.18 version.

Reply:
For now let v4 run 2nd and override.
we want truffle to test deployed code. not another contract.

Reply:
Ok.